run_tests.sh

```diff
@@ -2,6 +2,9 @@
 
 # php should be callable
 php -v &&
+# php should have speficic modules installed
+php -m | grep xdebug || exit 1 &&
+[ 70000 -le $(php -r "echo PHP_VERSION_ID;") ] || php -m | grep tideways || exit 1 &&
```

Comment:
```suggestion
[ 70000 -le $(php -r "echo PHP_VERSION_ID;") ] || php -m | grep tideways || exit 1 &&
php -m | grep mongo || exit 1 &&
```

Reply:
mongo is not installed anymore as it is not needed.